storybookjs/storybook
# Docs: Addon Docs - Update `transformSource` to `source.transform`

## What I did

`transformSource` was changed to `source.transform` in https://github.com/storybookjs/storybook/pull/21749 which renders this documentation stale. Noticed this while reading the `@storybook/addon-docs` documentation.

## Checklist for Contributors

### Testing

#### The changes in this PR are covered in the following automated tests:

#### Manual testing

Read the updated documentation and confirm that this is an accurate update (See https://github.com/storybookjs/storybook/pull/21749)

### Documentation



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

## Checklist for Maintainers

   <details>
     <summary>Available labels</summary>

     - `bug`: Internal changes that fixes incorrect behavior.
     - `maintenance`: User-facing maintenance tasks.
     - `dependencies`: Upgrading (sometimes downgrading) dependencies.
     - `build`: Internal-facing build tooling & test updates. Will not show up in release changelog.
     - `cleanup`: Minor cleanup style change. Will not show up in release changelog.
     - `documentation`: Documentation **only** changes. Will not show up in release changelog.
     - `feature request`: Introducing a new feature.
     - `BREAKING CHANGE`: Changes that break compatibility in some way with current major version.
     - `other`: Changes that don't fit in the above categories.
   
   </details>

### 🦋 Canary release



This PR does not have a canary release associated. You can request a canary release of this pull request by mentioning the `@storybookjs/core` team here.

_core team members can create a canary release [here](https://github.com/storybookjs/storybook/actions/workflows/canary-release-pr.yml) or locally with `gh workflow run --repo storybookjs/storybook canary-release-pr.yml --field pr=<PR_NUMBER>`_
